tests/main/default-tracks/task.yaml

```diff
@@ -35,10 +34,9 @@ execute: |
 
     snap remove --purge "$A_SNAP"
     snap remove --purge test-snapd-curl
-    snap remove --purge jq
 
     # now try a multi-install
-    snap install "$A_SNAP" "jq"
+    snap install "$A_SNAP" "test-snapd-tools"
```

Comment:
Hmm? Can you clarify what is going on here?

Reply:
It's simply using a different snap for install. There was no real reason to install jq in particular.